src/stacks-hierarchy/Root_ReceiptScreen.tsx

```diff
@@ -28,13 +29,20 @@ type MessageState =
 export function Root_ReceiptScreen({route}: Props) {
   const {orderId, orderVersion} = route.params;
   const styles = useStyles();
+  const {data, isSuccess, isLoading} = useProfileQuery();
   const [email, setEmail] = useState('');
   const [reference, setReference] = useState<string | undefined>(undefined);
   const [state, setState] = useState<MessageState>(undefined);
   const {t} = useTranslation();
   const a11yContext = useAccessibilityContext();
   const analytics = useAnalytics();
 
+  useEffect(() => {
+    if (isSuccess) {
+      setEmail(data.email);
```

Comment:
Should it be `data?.email`?

Reply:
Since we are checking isSuccess we know that we have received data from profile, and email is a mandatory field we don't need the ? 😄